awx/main/management/commands/callback_stats.py

```diff
@@ -0,0 +1,38 @@
+import time
+import sys
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        super(Command, self).__init__()
+        with connection.cursor() as cursor:
+            clear = False
+            while True:
+                lines = []
+                for relation in (
+                    'main_jobevent', 'main_inventoryupdateevent',
+                    'main_projectupdateevent', 'main_adhoccommandevent'
+                ):
+                    lines.append(relation)
+                    for label, interval in (
+                        ('last minute:   ', '1 minute'),
+                        ('last 5 minutes:', '5 minutes'),
+                        ('last hour:     ', '1 hour'),
+                    ):
+                        cursor.execute(
+                            f"SELECT MAX(id) - MIN(id) FROM {relation} WHERE created > now() - '{interval}'::interval;"
```

Comment:
With the latest change to how `created` is defined

https://github.com/ansible/awx/pull/5618/files#diff-f37b92a11438678a6a32ac23a7790f05R55

I'm not sure if this gives helpful data anymore. If the callback receiver is bottlenecked at any part of the system, then the events created in the last minute would be 0, no matter how fast the actual insertion rate is.

Reply:
Agreed, this should probably be `modified` instead.